internal/api/ui/login/renderer.go

```diff
@@ -315,13 +325,13 @@ func (l *Login) renderInternalError(w http.ResponseWriter, r *http.Request, auth
 	if err != nil {
 		_, msg = l.getErrorMessage(r, err)
 	}
-	data := l.getBaseData(r, authReq, "Error", "Internal", msg)
+	data := l.getBaseData(r, authReq, "Error","", "Internal", msg)
```

Comment:
if we change to i18n keys we either need one for `Error` or we should change to `Errors.Internal`

Reply:
changed to `Errors.Internal`